BlueprintUI/Sources/Element/ElementContent.swift

```diff
@@ -14,13 +14,17 @@ public struct ElementContent {
         configure(&builder)
         self.storage = builder
     }
+    
+    fileprivate init(storage : ContentStorage) {
+        self.storage = storage
+    }
 
     /// Measures the required size of this element's content.
     /// - Parameters:
     ///   - constraint: The size constraint.
     ///   - environment: The environment to measure in.
     /// - returns: The layout size needed by this content.
-    public func measure(in constraint: SizeConstraint, environment: Environment) -> CGSize {
+    public func measure(in constraint: SizeConstraint, environment: Environment = .empty) -> CGSize {
```

Comment:
I don't think this should have a default. It's not usually called with `.empty`, and this opens up a class of mistakes.

Reply:
Good point, removed